src/main/java/hudson/plugins/ec2/EC2OndemandSlave.java

```diff
@@ -456,7 +458,8 @@ public void terminate() {
                             Jenkins.get().removeNode(this);
                             LOGGER.info("Removed EC2 instance from jenkins controller: " + getInstanceId());
```

Comment:
Some of these `INFO` log messages could probably be replaced with messages to the `TaskListener`.